src/main/java/org/jenkinsci/plugins/github_branch_source/Connector.java

```diff
@@ -139,9 +156,9 @@ public static FormValidation checkScanCredentials(SCMSourceOwner context, String
     }
 
     @CheckForNull
-    public static StandardCredentials lookupScanCredentials(@CheckForNull SCMSourceOwner context,
-                                                                          @CheckForNull String apiUri,
-                                                                          @CheckForNull String scanCredentialsId) {
+    public static StandardCredentials lookupScanCredentials(@CheckForNull Item context,
```

Comment:
Please review and (if possible) merge #123 first, since this will clash with it.